src/api/app/jobs/worker_measurements_job.rb

```diff
@@ -50,4 +51,17 @@ def send_scheduler_metrics
       end
     end
   end
+
+  def send_daemon_metrics
+    running=0
+    nonrunning=0
+    @workerstatus.xpath('//partition/daemon').each do |daemon|
+      if daemon.attributes['state'].value == 'running'
+        running += 1
+      else
+        nonrunning += 1
+      end
+    end
+    RabbitmqBus.send_to_bus('metrics', "worker_status running=#{running},nonrunning=#{nonrunning}")
```

Comment:
Why summarize it in the metric already? Instead of individual measurements. Makes it hard to get more information, like *what* is `nonrunning` for instance.

Reply:
I rewrote it to collect more details